### ISSUE TITLE
Fast Clicking/Lag Allows Players To Level Up A Skill Past Cap

### ISSUE THREAD

**Issue by DistantDig** [Globox1997/LevelZ#271]
### Minecraft Version

1.19.x

### Mod Loader Version

0.14.19

### Mod Version

1.4.9

### Describe the Issue

We were working on a modpack when one of our playtesters realized he had a skill beyond the max level. Through further testing we found that we could reliably recreate the issue with an autoclicker, and we believe the first occurrence was due to lag (multiple clicks being processed quickly server side). These issues happened on a server if that helps at all. Thank you so much for continued work by the way, we really love the mod

### latest.log

At time of writing I do not have one, but if its completely necessary I can get one

### Do you use any other mods except the required ones?

We are loading a modpack, but it doesnt seem related to other mods and so we have not tested it.

**Comment by Globox1997** [comment]
Mh do you use the overall max level setting?

**Comment by DistantDig** [comment]
Yeah we do. We have our max level set to 60, and each skill seems to have a max of 20 but the skills can glitch past that cap
